amqp_mock/amqp_server/_amqp_server.py

```diff
@@ -44,6 +44,10 @@ def host(self) -> str:
     def port(self) -> int:
         return self._port
 
+    @port.setter
+    def port(self, value):
```

Comment:
```suggestion
    def port(self, value: int) -> None:
```

Reply:
Please run `make lint` to check types.

It will be run automatically in [CI](https://github.com/nikitanovosibirsk/amqp-mock/blob/master/.github/workflows/test.yml#L29) and it must pass for successful pipeline completion.